Sources/JWTKit/JWTKeyCollection.swift

```diff
@@ -38,7 +46,7 @@ public actor JWTKeyCollection: Sendable {
     func add(_ signer: JWTSigner, for kid: JWKIdentifier? = nil) -> Self {
         let signer = JWTSigner(algorithm: signer.algorithm, parser: signer.parser, serializer: signer.serializer)
 
-        if let kid = kid {
+        if let kid {
             if self.storage[kid] != nil {
                 print("Warning: Overwriting existing JWT signer for key identifier '\(kid)'.")
```

Comment:
Is there a `Logger` that can be used here instead of `print()`?

Reply:
Not really. Adding swift-log for that might not really be worth it

Reply:
Agreed, but at the same time IIRC it's considered bad form for libraries to `print()`. @0xTim Do you know what the recommendation is in such a case?

Reply:
Yeah printing is a no go, we need to use a logger if we want to log stuff. I'm surprised we haven't needed to up to this point